File: LeetCode-Programs/Largest Perimeter Triangle.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int per=0;
        for(int i=0;i<nums.size()-2;i++)
        {
            if(nums[i]+nums[i+1]>nums[i+2])
            {
                per=nums[i]+nums[i+1]+nums[i+2];
            }
        }
        return per;
    }
};

File: LeetCode-Programs/Largest Perimeter Triangle.py
class Solution:
    def largestPerimeter(self, nums: List[int]) -> int:
        '''nums.sort()
        if nums[0]+nums[1]>nums[2]:
            return nums[0]+nums[1]+nums[2]
        return 0
        '''
        nums.sort()
        per=0
        for i in range(len(nums)-2):
            if nums[i]+nums[i+1]>nums[i+2]:
                per=nums[i]+nums[i+1]+nums[i+2]
        return per
            
